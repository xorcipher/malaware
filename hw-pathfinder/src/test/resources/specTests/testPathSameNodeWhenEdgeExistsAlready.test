# Find a path with Dijkstra's algorithm when
# the nodes entered are the same node, and
# an edge already exists between the node and
# itself.

CreateGraph simpleGraph

AddNode simpleGraph A

AddEdge simpleGraph A A 5.0

FindPath simpleGraph A A