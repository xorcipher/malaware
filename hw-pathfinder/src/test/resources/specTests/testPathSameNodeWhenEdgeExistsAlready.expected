# Find a path with Dijkstra's algorithm when
# the nodes entered are the same node, and
# an edge already exists between the node and
# itself.

created graph simpleGraph

added node A to simpleGraph

added edge 5.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000